src/plugins/wb-data-scrub/wb-data-scrub.js

```diff
@@ -0,0 +1,145 @@
+							"yesBtn": "I agree",
+							"cancelBtn": "Go back to the form",
+							"viewMore": "Find out more"
+						},
+						fr: {
+							"msgboxHeader": "Supprimer les informations personnelles",
+							"msgBody1": "Les informations ajoutées dans les champs suivants sont identifiées comme des informations personnelles et seront soumises comme suit: ",
+							"msgBody3": "<p>La protection de la vie privée et des renseignements personnels représente un effort clé à travers tous nos services. Veuillez noter que les informations personnelles suivantes sont détachées de toutes les demandes de recherche avant toute activité de traitement:</p>",
+							"msgBody4": "<ul><li>8 chiffres ou plus séparés ou non par les caractères suivants : ' ', '/', '-', '.'</li><li>Numéro de série du passeport</li><li>E-mail</li><li>Code Postal</li><li>Des paramètres considérés comme non sécurisés dans une url</li></ul>",
+							"yesBtn": "Je suis d'accord",
+							"cancelBtn": "Retourner au formulaire",
+							"viewMore": "En savoir plus"
+						}
+					},
+					i18n = i18nDict[ wb.lang || "en" ],
+					moDal = document.createDocumentFragment(),
+					tpl = document.createElement( "div" ),
+					msgboxHeader = i18n.msgboxHeader,
+					yesBtn = i18n.yesBtn,
+					cancelBtn = i18n.cancelBtn,
+					msgBody1 = i18n.msgBody1,
+					msgBody2 = "<ul>",
+					msgBody3 = i18n.msgBody3,
+					msgBody4 = i18n.msgBody4,
+					viewMore = i18n.viewMore,
+					countPiifield = 0,
+					checkFormValues = function( scrubData ) {
+
+						// identify form elements that were assigned to be scrubbed
+						$( selector, form ).each( function() {
+							var $elmInput = $( this ),
+								elmInputVal = $elmInput.val(),
+								scrubbedVal = wb.findPotentialPII( elmInputVal, true );
+							$elmInput.removeAttr( "data-first-to-scrub" );
+
+							// if the helper find any PII values clean the PII and assign new value to the form element with the output string
+							if ( scrubbedVal !== false ) {
+								countPiifield++;
+								if ( countPiifield === 1 ) {
+									$elmInput.focus();
+								}
+								msgBody2 = msgBody2 + "<li>" + $elmInput.prev( "label[for=" + $elmInput.attr( "id" ) + "]" ).text() + ": " + ( scrubbedVal !== "" ? scrubbedVal : "" ) + "</li>";
+								isPii = true;
+								if ( scrubData ) {
+									$elmInput.val( scrubbedVal === false ? elmInputVal : scrubbedVal );
+									isPii = false;
+								}
+							}
+						} );
+						msgBody2 += "</ul>";
+					};
+				if ( document.getElementById( moDalId ) ) {
+					document.getElementById( moDalId ).remove();
+				}
+				checkFormValues();
+				tpl.innerHTML = "<section id='" + moDalId + "' " + "class='modal-dialog modal-content overlay-def'>" +
+										"<header class='modal-header'><h2 class='modal-title'>" + msgboxHeader + "</h2></header>" +
+										"<div class='modal-body'>" +
+										"<p>" + msgBody1 + "</p>" +
+										"<p>" + msgBody2 + "</p>" +
```

Comment:
2 paragraph? Can we stick to 1 if possible. When following the example the second paragraph become empty

Reply:
the 2 paragraph are merged and an eslint exception added, as discussed